### PR TITLE
#431 integration test of changes made to a judgment - search

### PR DESCRIPTION
**Pull request by madryk**
see #431

### CODE REVIEW

File: saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+public class TestInMemoryNacObjectFactory {
+
+    
+    //------------------------ LOGIC --------------------------
+    /**
+     * Creates {@link NationalAppealChamberJudgment} hierarchy
+     * with default field data.
+     * @return NationalAppealChamberJudgment.
+     */
+    static NationalAppealChamberJudgment createNacJudgment(){
```
Comment (pawelczak) [inline saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java]: I think you should put `public` here, just not to confuse others.

File: saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        return nacJudgment;
+    }
+
+
+    /**
+     * Creates {@link NationalAppealChamberJudgment} with minimal set of fields (necessaries for storing in db)
+     * filled with random data.
+     * @return NationalAppealChamberJudgment
+     */
+    static NationalAppealChamberJudgment createSimpleNacJudgment(){
```
Comment (pawelczak) [inline saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java]: `public`

File: saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+        return judgment;
+    }
+
+    /**
+     * Creates list of {@link NationalAppealChamberJudgment} with fields filled with random values.
+     * @param size of the list.
+     * @return list of NationalAppealChamberJudgment
+     */
+    static List<NationalAppealChamberJudgment> createNacJudgmentListWithRandomData(int size){
```
Comment (pawelczak) [inline saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java]: `public`

Comment (madryk) [inline saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java]: ok

File: saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java (hunk 2 above)
Comment (madryk) [inline saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java]: ok

File: saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java (hunk 1 above)
Comment (madryk) [inline saos-persistence/src/test/java/pl/edu/icm/saos/persistence/common/TestInMemoryNacObjectFactory.java]: ok

**Pull request closed without merging**